src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/dto/samm/ProductStockSammDto.java

```diff
@@ -51,7 +51,7 @@ public class ProductStockSammDto extends MessageContentDto {
     @Pattern(regexp = "(^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)|(^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)")
     private Optional<String> materialNumberCatenaX;
 
-    private Optional<String> materialNumberSupplier;
+    //private Optional<String> materialNumberSupplier;
```

Comment:
We need to reinclude this. This may lead to serialization issues.

Reply:
Done